starcompactor/extractors/mysql.py

```diff
@@ -69,15 +67,15 @@ def traces_query(db, start=None, end=None):
     if conditionals:
         sql = '{} AND {}'.format(sql, ' AND '.join(conditionals))
 
-    sql += ' ORDER BY ia.created_at'
     return db.query(sql, args=params, limit=None)
 
-
-def traces(db, start=None, end=None):
-    results = traces_query(db, start, end)
+def traces(db, database, start=None, end=None):
+    results = traces_query(db, database, start, end)
 
     for event in results:
-        if event['finish_time'] is None:
+        for key in event:
+            if key in TRACE_EVENT_KEY_RENAME_MAP.keys(): event[TRACE_EVENT_KEY_RENAME_MAP[key]] = event.pop(key)
+        if event['FINISH_TIME'] is None:
```

Comment:
A bit more expressive: (nitpick)

```suggestion
        if not event['FINISH_TIME']:
```